packages/features/ee/workflows/lib/reminders/templates/emailReminderTemplate.ts

```diff
@@ -1,43 +1,50 @@
+import { WorkflowActions } from "@prisma/client";
+
 import dayjs from "@calcom/dayjs";
 import { APP_NAME } from "@calcom/lib/constants";
 
 const emailReminderTemplate = (
-  startTime: string,
-  endTime: string,
-  eventName: string,
-  timeZone: string,
-  attendee: string,
-  name: string
+  isEditingMode: boolean,
+  action?: WorkflowActions,
+  startTime?: string,
+  endTime?: string,
+  eventName?: string,
+  timeZone?: string,
+  attendee?: string,
+  name?: string,
+  isBrandingDisabled?: boolean
 ) => {
-  const emailSubject = `Reminder: ${eventName} - ${dayjs(startTime)
-    .tz(timeZone)
-    .format("ddd, MMM D, YYYY")} ${dayjs(startTime).tz(timeZone).format("H:mmA")}`;
-
-  const templateBodyText = `Hi ${name},this is a reminder that your meeting (${eventName}) with ${attendee} is on ${dayjs(
-    startTime
-  )
-    .tz(timeZone)
-    .format("YYYY MMM D")} at ${dayjs(startTime).tz(timeZone).format("h:mmA")} ${timeZone}.`;
+  let eventDate = "";
+
+  if (isEditingMode) {
+    endTime = "{EVENT_END_TIME}";
+    eventName = "{EVENT_NAME}";
+    timeZone = "{TIMEZONE}";
+    attendee = action === WorkflowActions.EMAIL_ATTENDEE ? "{ORGANIZER}" : "{ATTENDEE}";
+    name = action === WorkflowActions.EMAIL_ATTENDEE ? "{ATTENDEE}" : "{ORGANIZER}";
+    eventDate = "{EVENT_DATE_ddd, MMM D, YYYY H:mmA}";
+  } else {
+    eventDate = dayjs(startTime).tz(timeZone).format("ddd, MMM D, YYYY H:mmA");
+
+    endTime = dayjs(endTime).tz(timeZone).format("H:mmA");
+  }
+  const emailSubject = `Reminder: ${eventName} - ${eventDate}`;
 
   const introHtml = `<body>Hi${
     name ? " " + name : ""
   },<br><br>This is a reminder about your upcoming event.<br><br>`;
 
-  const eventHtml = `<div style="font-weight: bold;">Event:</div>${eventName}<br><br>`;
+  const eventHtml = `<div><strong class="editor-text-bold">Event:</strong></div>${eventName}<br><br>`;
 
-  const dateTimeHtml = `<div style="font-weight: bold;">Date & Time:</div>${dayjs(startTime)
-    .tz(timeZone)
-    .format("ddd, MMM D, YYYY H:mmA")} - ${dayjs(endTime)
-    .tz(timeZone)
-    .format("H:mmA")} (${timeZone})<br><br>`;
+  const dateTimeHtml = `<div><strong class="editor-text-bold">Date & Time:</strong></div>${eventDate} - ${endTime} (${timeZone})<br><br>`;
 
-  const attendeeHtml = `<div style="font-weight: bold;">Attendees:</div>You & ${attendee}<br><br>`;
+  const attendeeHtml = `<div><strong class="editor-text-bold">Attendees:</strong></div>You & ${attendee}<br><br>`;
 
-  const endingHtml = `This reminder was triggered by a Workflow in Cal.<br><br>_<br><br>Scheduling by ${APP_NAME}</body>`;
+  const branding = !isBrandingDisabled && !isEditingMode ? `<br><br>_<br><br>Scheduling by ${APP_NAME}` : "";
```

Comment:
I am not sure if we should send this branding text in emails from self-hosted instances. Though it would have their APP_NAME in the text but suddenly starting sending this text might be unexpected.

Reply:
Also, we are not showing this in the template during editing but maybe we should mention somewhere(in the app) that there would be branding text as well in all emails depending on the connected event-type's `hideBranding` status.